privacyidea/lib/eventhandler/tokenhandler.py

```diff
@@ -147,6 +150,17 @@ def actions(cls):
                             "description": _("Dynamically read the mobile number "
                                              "from the user store.")
                         },
+                        "dynamic_email": {
+                            "type": "bool",
+                            "visibleIf": "tokentype",
+                            "visibleValue": "email",
+                            "description": _("Dynamically reag the email address "
+                                             "from the user store.")
+                        },
+                        "additional_params": {
+                            "type": "str",
+                            "description": _("A dictionary of additional init parameters.")
```

Comment:
Maybe add to description that this should be in yaml-format? or the docs?

Reply:
I think I will add it to the docs. After all, we need a dictionary from a logical standpoint. 
``"[1,2,3,4]"`` would be a valid yaml format but would result in an error.
So in fact it is a dictionary - this is why I prefer not to tell, that we internally use a yaml parser (just to avoid confusions)

Reply:
done